src/keria/core/longrunning.py

```diff
@@ -412,22 +488,43 @@ def status(self, op):
             else:
                 operation.done = False
 
-        elif op.type in (OpTypes.done, ):
+        elif op.type in (OpTypes.submit,):
```

Comment:
Add longrunning logic for submit operation. If we have the kever, have no submits, and have no witnessReceiptor doers then we are done.